eclair-core/src/main/scala/fr/acinq/eclair/Eclair.scala

```diff
@@ -144,19 +145,19 @@ class EclairImpl(appKit: Kit) extends Eclair {
       pushMsat = pushAmount_opt.getOrElse(0 msat),
       fundingTxFeeratePerKw_opt = fundingFeerateSatByte_opt.map(feerateByte2Kw),
       channelFlags = flags_opt.map(_.toByte),
-      timeout_opt = Some(openTimeout))).mapTo[String]
+      timeout_opt = Some(openTimeout))).mapTo[ChannelCommandResponse].map(_.toString)
```

Comment:
You could return a `ChannelCommandResponse` and define a json deserializer that does a `toString`. Not sure if that'd be better.

Reply:
It probably makes sense to return a `ChannelCommandResponse` and push the conversion to string (which is lossy) closer to the boundary of the program.

Reply:
Yes. The disadvantage being that there is a symmetry problem given that this function doesn't take a `Command`. I believe we are in bikeshed territory.

Reply:
True, but I think it's quite ok there :)